.swiftlint.yml

```diff
@@ -1,12 +1,76 @@
+#
+# Automatically downloaded from https://raw.githubusercontent.com/raywenderlich/swift-style-guide/master/com.raywenderlich.swiftlint.yml by SwiftLint on 22/08/2022 at 4:40:02 pm.
+#
+
+parent_config: https://raw.githubusercontent.com/raywenderlich/swift-style-guide/master/com.raywenderlich.swiftlint.yml
+
 included: # paths to include during linting. `--path` is ignored if present.
   - Source
-  - iOS Example
   - iOS Example Frame
   - iOS Example Frame Carthage
   - iOS Example Frame SPM
-excluded: # paths to ignore during linting. Takes precedence over `included`.
+  - Tests
+  - scripts
+  - Documentation
+
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - iOS Example Frame/Pods
   - Carthage
-  - Pods
-identifier_name:
-  excluded:
-    - id
+  - Checkout
+  - iOS Example Apple Pay
+  - iOS Example
+
+disabled_rules:
+  - discarded_notification_center_observer
+  - notification_center_detachment
+  - orphaned_doc_comment
+  - todo
+  - unused_capture_list
+  - switch_case_alignment
+  - multiline_arguments
+  - multiline_parameters_brackets
+  - multiline_literal_brackets
+  - class_delegate_protocol
+
+attributes:
+  always_on_same_line:
+    - "@IBSegueAction"
+    - "@IBAction"
+    - "@NSManaged"
+    - "@objc"
+    - "@frozen"
+
+indentation_width:
+  indentation_width: 4
+  ignores_function_declarations: true
+  ignores_comments: true
+
+line_length:
+  warning: 140
+  ignores_urls: true
+  ignores_function_declarations: true
+  ignores_comments: true
+
+trailing_whitespace:
+  ignores_empty_lines: false
+  ignores_comments: true
+  max_empty_lines: 2
+
+vertical_whitespace:
+  max_empty_lines: 2
+
+cyclomatic_complexity:
+    warning: 12
+    error: 15
+
+nesting:
+  type_level:
+    warning: 3
+
+  function_level:
+    warning: 3
+
+force_unwrapping: warning
```

Comment:
I think force_unwrapping should always be an error. Usually main impact from this is just ignoring unit test modules from being linted, but I wonder if that's a price worth paying?

Reply:
This should be error and we ignore unit tests and other places ( file and line level swiftlint:disable) where it makes sense. Me and @ehab-al-cko discussed about this and we need to find other things that can be errors so we accidentally don't dont misuse swift lint as currently everything is a warning. ( we need some enforcing rules )

Reply:
Also to add I mentioned @ehab-al-cko to make this as a warning for now and create a separate ticket to change this to error and fix issues & disable line level and file level in a separate PR for clarity.

Reply:
All are fixed with unit test too, i will change it error.